jenkinsapi_tests/unittests/test_build.py

```diff
@@ -49,6 +85,12 @@ def test_name(build):
     assert build.name == 'foo #1'
 
 
+def test_name_build_with_diff_baseurl(build_with_diff_baseurl):
```

Comment:
are we testing the fact that jenkins is (mis)configured with one url and job will build will have another url? Or we just testing that build can be created at all? I would assume that we are testing url related change...